x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -1233,7 +1233,6 @@
     "home.addData.metrics.nameTitle": "メトリック",
     "home.addData.sampleDataLink": "データセットと Kibana ダッシュボードを読み込む",
     "home.addData.sampleDataTitle": "サンプルデータの追加",
-    "home.addData.siem.addSiemEventsButtonLabel": "イベントを追加",
```

Comment:
do not delete just rename the key to `home.addData.security.addSecurityEventsButtonLabel`